R/git.R

```diff
@@ -132,6 +132,102 @@ use_git_config <- function(scope = c("user", "project"), ...) {
   }
 }
 
+#' Choose git protocol and credential defaults
```

Comment:
Would this be better as "choose default git protocol"? Shouldn't it only apply to new repositories that don't already have a protocol?

Reply:
Sure ok. Done in master.